content/_data/changelogs/weekly.yml

```diff
@@ -298,6 +298,7 @@
     - type: major rfe
       message: >
         <strong>Jenkins now requires Java 8 to run.</strong>
+         Note this applies to "agents" too (you can configure a custom <code>JavaPath</code> in their "Advanced" properties)!
```

Comment:
Well, the issue *we* struck was that for some other things running on those hosts, a different Java version was the system default. So we installed a separate JAVA_HOME with Java8 just for running the agent.